AI/VCAI/VCAI.cpp

```diff
@@ -3325,30 +3338,30 @@ this functions returns one target tile or invalid tile. We will use it to poll p
 For ship construction etc, another function (goal?) is needed
 */
 {
-	int3 ret(-1,-1,-1);
+	int3 ret(-1, -1, -1);
 
 	int sourceSector = retreiveTile(h->visitablePos()),
-		destinationSector = retreiveTile(dst);
+	destinationSector = retreiveTile(dst);
 
-	const Sector *src = &infoOnSectors[sourceSector],
-		*dest = &infoOnSectors[destinationSector];
+	const Sector * src = &infoOnSectors[sourceSector],
+	* dest = &infoOnSectors[destinationSector];

```

Comment:
Just in case this one is inteded because there is no difference for uncrustify config between:
```
const Sector * src = &infoOnSectors[sourceSector],
* dest = &infoOnSectors[destinationSector];
```
Or:
```
const Sector * src = &infoOnSectors[sourceSector];
const Sector * dest = &infoOnSectors[destinationSector];
```